webpack/scenes/AlternateContentSources/Create/Steps/SelectSource.js

```diff
@@ -19,10 +24,10 @@ const SelectSource = () => {
     }
   };
 
-  const typeOptions = [
-    { value: 'yum', label: __('Yum') },
-    { value: 'file', label: __('File') },
-  ];
+  const typeOptions = [{ value: 'yum', label: __('Yum') }];
+  if (acsType !== 'rhui') {
```

Comment:
I think we could actually hide the content type selection entirely since Yum will be the only option.